famapy/metamodels/fm_metamodel/models/feature_model.py

```diff
@@ -166,11 +169,6 @@ def get_features(self) -> list['Feature']:
     def get_constraints(self) -> list['Constraint']:
         return self.ctcs
 
-    def get_feature_by_name(self, feature_name: str) -> 'Feature':
```

Comment:
Why this mehod has been deleted. Did you crosscrecked if it's used by other plugins?

Reply:
The implementation of the method is wrong, the attribute `features_by_name` was removed within the last update because of the inconsistency with the cached list of features. Therefore, the current method will raise an error if used. I think this is also part of Issue #27